crates/ruma-common/src/identifiers.rs

```diff
@@ -31,7 +31,7 @@ pub use self::{
     user_id::UserId,
 };
 #[doc(inline)]
-pub use ruma_identifiers_validation::error::Error;
+pub use ruma_identifiers_validation::error::Error as InvalidId;
```

Comment:
Shouldn't we keep the `Error` suffix? I believe that's what we do in other crates.

Reply:
I think in the grand scheme of things this error type name isn't unusual, and I prefer it over `InvalidIdError` because that seems weirdly redundant. `IdParseError` could work, but I kinda like the `InvalidId` name.

I'm curious what other contributors think.

Reply:
`IdParseError` sounds more "correct" than `InvalidId`, as the former describes what happens, while the other prescribes what the caller did wrong. I prefer the first one on that basis imo.